libcontainer/console.go

```diff
@@ -11,5 +17,59 @@ type Console interface {
 	Path() string
 
 	// Fd returns the fd for the master of the pty.
-	Fd() uintptr
+	File() *os.File
+}
+
+const (
+	TerminalInfoVersion = 1
+	TerminalInfoType    = "terminal"
+)
+
+// TerminalInfo is the structure which is passed as the nonancilliary data
```

Comment:
Or maybe wiktionary is funded by the UE people who deny the superiority of BE. :wink: